.github/workflows/branch-build.yml

```
@@ -75,34 +85,31 @@ jobs:
       - name: Download k3d
         run: "wget -q -O - https://raw.githubusercontent.com/k3d-io/k3d/main/install.sh | TAG=${K3D_VERSION} bash"
 
-      - name: Create cluster
-        run: "k3d cluster create botkube --wait --timeout=5m"
+      - name: Create cluster to test ${{ matrix.integration }}
+        run: "k3d cluster create ${{ matrix.integration }}-test-cluster --wait --timeout=5m"
 
-      - name: Install Botkube
+      - name: Install BotKube to test ${{ matrix.integration }}
```

As the rebranding was already applied, please ensure proper casing 🙂 
```suggestion
      - name: Install Botkube to test ${{ matrix.integration }}
```

---

Done. There was another one buried in a comment.